pyteal/ast/subroutine_test.py

```diff
@@ -171,7 +299,15 @@ def fnReturningExprSubclass(a: ScratchVar, b: Expr) -> Return:
         return Return()
 
     def fnWithMixedAnns4AndBytesReturn(a: Expr, b: ScratchVar) -> Bytes:
-        return Bytes("helo")
+        return Bytes("hello uwu")
+
+    def fnWithMixedAnnsABIRet1(
+        a: Expr, b: ScratchVar, c: abi.Uint16
+    ) -> abi.StaticArray[abi.Uint32, Literal[10]]:
```

Comment:
Ok, I see the my preferred API is available, yay! But I'm still lobying to _disallow_ the other approach. I.e., don't allow `output` as a special keyword only variable.

Reply:
Woops. I spoke too soon. This is a _negative_ example. Using to assert that an error has occurred